src/primitives/block.h

```diff
@@ -25,7 +39,7 @@ class CBlockHeader
 public:
     static const int32_t CURRENT_VERSION=SIZE_FORK_VERSION;
     // This code knows the rules for these block versions:
-    static const int32_t UNDERSTOOD_VERSIONS = (SIZE_FORK_VERSION | 3 | 2 | 1);
+    static const int32_t UNDERSTOOD_VERSIONS = (SIZE_FORK_VERSION | 4 | 3 | 2 | 1);
```

Comment:
What's up with the strange bitwise OR'ing?
4 | 3 | 2 | 1 is the same as 4 | 2 | 1 (it's both 7)
The extra 3 there is meaningless, isn't it?